packages/connection/src/class-manager.php

```diff
@@ -989,7 +989,7 @@ protected function validate_remote_register_response( $response ) {
 			$registration_response = false;
 		}
 
-		$code_type = intval( $code / 100 );
+		$code_type = (int) ($code / 100);
```

Comment:
We'll need spaces inside the parenthesises:

```suggestion
		$code_type = (int) ( $code / 100 );
```